js/scripts/conversation_loaders/village_conversation_loader.js

```diff
@@ -256,4 +264,35 @@ const minerConversationLoader = () => {
 
         ]
     };
-};
\ No newline at end of file
+};
+
+const wizardConversationLoader = () => {
+    const wiz = DialogBubble.SPEAKERS.WIZARD;
+    const chad = DialogBubble.SPEAKERS.CHAD;
+    const none = DialogBubble.SPEAKERS.NONE;
+    return {
+        threateningIntroduction: [
+            new DialogBubble(wiz, 
+                "Stop right there, boy!"),
```

Comment:
Can we give him the voice of Kratos? That would be phenomenal.

Reply:
He's way too cowardly for that!